geoips/plugins/modules/readers/utils/hrit_reader.py

```diff
@@ -886,10 +888,10 @@ def __read_time_cds(self, expanded=False):
         millisec = self.__rf(">u4")
         if expanded:
             microsec = self.__rf(">u2")
-            nanosec = self.__rf(">u2")
+            # nanosec = self.__rf(">u2")
         else:
             microsec = 0
-            nanosec = 0
+            # nanosec = 0
```

Comment:
This is not actually reading anything?  Just casting an existing variable as a u2 int? @jsolbrig